plugins/modules/site_workflow_manager.py

```diff
@@ -882,9 +973,8 @@ def is_floor_updated(self, updated_site, requested_site):
             It checks if the name, rf_model, length, width, and height are equal, indicating
             that the floor details have been updated. Returns True if the details match, and False otherwise.
         """
-
```

Comment:
self.log("Starting floor update check with updated_site: {} and requested_site: {}".format(updated_site, requested_site), "DEBUG")

Reply:
Added this log